publishable/config/voyager.php

```diff
@@ -94,21 +95,19 @@
     */
 
     'widgets' => [
-        [
-            'name'  => 'User',
+        'user' => [
             'icon'  => 'voyager-group',
             'model' => TCG\Voyager\Models\User::class,
             'url'   => 'admin/users',
             'image' => '/images/widget-backgrounds/02.png',
         ],
```

Comment:
yeah i saw the new changes, am not sure whats ur plan for widgets but having only the classes in the config is not right, u may even just remove it all together and enforce it through the class, but what if the user wants to make some changes ?

also widgets is easier to understand than dimmer, maybe instead of changing the name we can have a slight more detailed description about what they are for.